src/pages/filter/ui/filter-refiner-groups.tsx

```diff
@@ -41,7 +42,7 @@ export const FilterRefinerGroups = observer((): ReactElement => {
         [
           FilterKindEnum.Enum,
           name,
-          '',
```

Comment:
ConditionJoinMode.OR is 'OR' string, but you replace an empty string with it. Is there no mistake here?